### DISCUSSION TITLE
Value from different strings

### DISCUSSION THREAD

**Discussion by cirinx** [djansen1987/SAJeSolar#61] [Ideas]
Hi, Is it possible to find out the value of PV Voltage/Current/Power1, PV Voltage/Current/Power2, PV Voltage/Current/Power3, from the different strings of the plant  ?
I found it on the esolar Portal.
Thank you.

**Comment by djansen1987** [comment]
HI, this is a difficult task. As the api only provides a large dataset with all historical data. If a PR is created for it i am happy to integrate it.
